doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -0,0 +1,31 @@
+= Transforming coordinate systems
```

Comment:
```suggestion
= Coordinate Transformations
```

Reply:
I would stick to a more active tone, similar to "Modelling a traffic participant".